nss-day-cohort-43-csharp/Fullstack--span--Halen
# Fm read time

# Description
Users should see a welcome page when they sign into the site. User should see a read time on a post 
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Please make sure you see a welcome page and a read time on the post. If it's one minute should display 1 minute, otherwise should display minutes 
# Checklist:
